project/Version.sc

```diff
@@ -0,0 +1,17 @@
+object SpinalVersion {
+  val compilers = List("2.11.12", "2.12.13", "2.13.6")
+  val compilerIsRC = false
+
+  val isDev = true
+  val isSnapshot = false
+  private def snapshot = if (isSnapshot) "-SNAPSHOT" else ""
+  private val major = "1.7.2"
+  val all         = if(isDev) "dev" else s"$major$snapshot"
+  val sim         = all
+  val core        = all
+  val lib         = all
+  val ip          = all
+  val debugger    = all
+  val demo        = all
+  val tester      = all
+}
```

Comment:
Isn't this just a copy of project/Version.scala?

Reply:
> Isn't this just a copy of project/Version.scala?

Yes, it's not done yet. I want to use a symbol link instead. 
However, it is not that friendly to Windows. I am finding the way.

Reply:
Can it not load the existing Version file as-is?

Reply:
> Can it not load the existing Version file as-is?

To my knowledge, the mill would load only file with extension 'sc' through it's "import $file" syntax.